src/controllers/accounts.js

```diff
@@ -14,6 +14,13 @@ exports.getCollection = function * find () {
   this.body = _.invoke(accounts, 'getDataExternal')
 }
 
+exports.getConnectors = function * () {
+  const accounts = yield Account.findAll({
+    where: { connector: { $ne: null } }
+  })
+  this.body = _.invoke(accounts, 'getDataConnector')
```

Comment:
We have previously talked about a naming scheme for these controller methods:

For a controller called `quibbles.js` and a model called `Quibble`:

`getResource` - Get a Quibble by primary key, usually GET /quibbles/:id
`getResourceBy[...]` - Get a Quibble by some other key, usually GET /quibbles/something/:foo
`getCollection` - Get a bunch of Quibbles, usually GET /quibbles
`get[...]Resource` - Get a resource related to Quibble by the Quibble's primary key, usually GET /quibbles/:id/something
`get[...]ResourceBy[...]` - Get a resource related to Quibble by some other index, usually GET /quibbles/something/:foo/something

So this could be `getConnectorCollection` perhaps.

We never really standardized on this scheme though and I don't care enough to worry about this. This name is only used in one place internally in the same project, so it's trivial to change anytime we want.